src/Medusa.ts

```diff
@@ -3,23 +3,23 @@ import { getConfigFile } from 'medusa-core-utils/dist';
 import * as getEndpoints from 'express-list-endpoints';
 import { Express } from 'express';
 import { AwilixContainer } from 'awilix';
-import { Constructor, metadataReader, Logger } from './core';
+import { Constructor, Logger, metadataReader } from './core';
 import {
-	adminApiLoader,
-	databaseLoader,
-	migrationsLoader,
-	overrideEntitiesLoader,
-	overrideRepositoriesLoader,
-	pluginsLoadersAndListeners,
-	servicesLoader,
-	storeApiLoader,
-	validatorsLoader,
+    adminApiLoader,
+    databaseLoader,
+    migrationsLoader,
+    overrideEntitiesLoader,
+    overrideRepositoriesLoader,
+    pluginsLoadersAndListeners,
+    servicesLoader,
+    storeApiLoader,
```

Comment:
Codacy has a fix for the issue: [Replace `····` with `↹`](https://app.codacy.com/gh/adrien2p/medusa-extender/pullRequest?prid=9061797)
```suggestion
	storeApiLoader,
```